src/plugin/indexer-elastic/README.md

```diff
@@ -32,10 +32,12 @@ Parameter Name | Description | Default value
 --|--|--
 host | Comma-separated list of hostnames to send documents to using [TransportClient](https://static.javadoc.io/org.elasticsearch/elasticsearch/5.3.0/org/elasticsearch/client/transport/TransportClient.html). Either host and port must be defined or cluster. | 
```

Comment:
I missed that the `or cluster` part of the sentence is now obsolete (due to the removal) but that is not a blocker from my side. If you fix it before merging it would be great.

Reply:
Hi, kind of missed this one. I removed this in the latest commit.